app/TDG/ReservationTDG.py

```diff
@@ -1,110 +1,122 @@
-#ReservationTDG
+# ReservationTDG
 from app.db import connect_db
 
+
 def find(id):
-    conn = connect_db()
-    if conn:
-        cur = conn.cursor()
-        cur.execute("""SELECT * FROM reservationTable WHERE reservationId = %s;""", (id,))
-        data = cur.fetchall()
-        conn.close()
-        return data
-    else:
-        return []
-
-def insert(id, room, description, holder, timeslot):
-    conn = connect_db()
-    if conn:
-        cur = conn.cursor()
-        cur.execute("""INSERT INTO reservationTable(reservationId, room, description, holder, timeslot) VALUES
-                (%s, %s, %s, %s, %s);""", (id, room, description, holder, timeslot))
-        conn.commit()
-        conn.close()
+	conn = connect_db()
+	if conn:
+		cur = conn.cursor()
+		cur.execute("""SELECT * FROM reservationTable WHERE reservationId = %s;""", (id,))
+		data = cur.fetchall()
+		conn.close()
+		return data
+	else:
+		return []
+
+
+def insert(id, room, description, holder, timeslot, equipment):
+	conn = connect_db()
+	if conn:
+		cur = conn.cursor()
+		print(timeslot)
+		cur.execute("""INSERT INTO reservationTable(reservationId, room, description, holder, timeslot, equipment) VALUES
+                (%s, %s, %s, %s, %s, %s);""", (id, room, description, holder, timeslot, equipment))
+		conn.commit()
+		conn.close()
+
 
 def update(id, roomId, userId, description, timeslot):
```

Comment:
I might change this one in the future to allow updating of equipment...? May not be necessary.